ReactCommon/react/renderer/scheduler/Scheduler.cpp

```diff
@@ -351,6 +351,10 @@ ContextContainer::Shared Scheduler::getContextContainer() const {
   return contextContainer_;
 }
 
+std::shared_ptr<UIManager> Scheduler::getUIManager() const {
+  return uiManager_;
```

Comment:
(I know this is a cherry pick)
Why isn't this `_uiManager"`...

Reply:
> _uiManager

It is C++ and we suffix with ```_``` there.
In ObjC we would prefix with ```_```

Reply:
TIL :)